### PR TITLE
Fix some errors related to conversion to HAR

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

File: agent/pkg/utils/har.go [hunk 1]
```diff
@@ -203,9 +202,11 @@ func NewResponse(response *api.GenericMessage) (harResponse *har.Response, err e
 
 	status := int(resDetails["status"].(float64))
 	if strings.HasPrefix(mimeType.(string), "application/grpc") {
-		status, err = strconv.Atoi(_status)
+		if _status != "" {
```
Comment (gadotroee) [inline agent/pkg/utils/har.go:205]: why it is happen in GRPC and not before when we converting to int? (status := int(resDetails["status"].(float64)))

**Review by gadotroee — COMMENTED** [review]
small comment (question)

File: agent/pkg/utils/har.go (hunk 1 above)
Comment (mertyildiran) [reply agent/pkg/utils/har.go:205]: @gadotroee the variable name is `_status` and not `status`. It's different, it comes from `:status` header.

Comment (mertyildiran) [reply agent/pkg/utils/har.go]: @gadotroee OK fixed the wrong variable name in `BuildHeaders` function.

File: agent/pkg/utils/har.go [hunk 2]
```diff
@@ -205,7 +204,7 @@ func NewResponse(response *api.GenericMessage) (harResponse *har.Response, err e
 	if strings.HasPrefix(mimeType.(string), "application/grpc") {
 		status, err = strconv.Atoi(_status)
 		if err != nil {
-			tap.SilentError("convert-response-status-for-har", "Failed converting status to int %s (%v,%+v)", err, err, err)
+			rlog.Debugf("Failed converting status to int %s (%v,%+v)", err, err, err)
```
Comment (gadotroee) [inline agent/pkg/utils/har.go:207]: change to error?

Comment (mertyildiran) [reply agent/pkg/utils/har.go:207]: @gadotroee I didn't understand.

Comment (mertyildiran) [reply agent/pkg/utils/har.go]: @gadotroee oh OK `Errorf` done.

**Review by gadotroee — APPROVED** [review]

**Pull request merged**